FirebaseRemoteConfig/Sources/RCNFetch.m

```diff
@@ -476,11 +490,12 @@ - (void)fetchWithUserProperties:(NSDictionary *)userProperties
 
       // We had a successful fetch. Update the current eTag in settings if different.
       NSString *latestETag = ((NSHTTPURLResponse *)response).allHeaderFields[kETagHeaderName];
-      if (!self->_settings.lastETag || !([self->_settings.lastETag isEqualToString:latestETag])) {
-        self->_settings.lastETag = latestETag;
+      if (!strongSelf->_settings.lastETag ||
+          !([self->_settings.lastETag isEqualToString:latestETag])) {
```

Comment:
This looks good! Just noticed this though, should the `self` here be a `strongSelf`?

Reply:
Good catch! Yeah, ideally this code should be refactored to improve readability.